pkg/apis/ark/v1/config.go

```diff
@@ -98,6 +98,7 @@ type AWSConfig struct {
 	DisableSSL       bool   `json:"disableSSL"`
 	S3ForcePathStyle bool   `json:"s3ForcePathStyle"`
 	S3Url            string `json:"s3Url"`
+	KMSKeyID         string `json:"kmsKeyId"`
```

Comment:
I got tripped up on this being `kmsKeyId` and not `kmsKeyID`. It looks like we're inconsistent in other keys (`s3Url` rather than `s3URL` but `disableSSL`, not `disableSsl`). @ncdc thoughts?